src/elements/content-sidebar/activity-feed/task-new/Task.scss

```diff
@@ -55,8 +55,8 @@ $error-color: $red;
         width: 32px;
 
         // select all but first instance
-        & + & {
```

Comment:
& + & doesn't work here in this case since .bcs-task-assignment-avatar is nested